Mathlib/Tactic/Linter/UnicodeLinter.lean

```diff
@@ -0,0 +1,36 @@
+/-
+Copyright (c) 2024 Jon Eugster. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Adomas Baliuka, Jon Eugster
+-/
+
+import Mathlib.Init
+
+/-!
+
+# Tools for Unicode Linter
+
+The actual linter is defined in `TextBased.lean`.
+
+This file provides all white-lists and other tools used by the linter.
```

Comment:
Currently, there is no whitelist in this file (but merely a blacklist). In addition, I've come to prefer blocklist and allowlist --- this is technically often better and avoids black-/white connotations which *can* be problematic.
```suggestion
This file defines the blocklist and other tools used by the linter.